client/README.md

```diff
@@ -1,3 +1,23 @@
 ## Client
 
 This is the place for your application front-end files.
+
+Instructions:
+
+1) Make sure the `DB_URL` environment variable is set
+
+2) Type the following from toplevel dir
+
+```
+npm install strongloop
```

Comment:
`npm install` should be sufficient since strongloop is listed inside of package.json (note: strongloop is not the only dependency so it's perhaps even advisable for dev usage to do `npm install --dev`)